src/components/PDFView/index.native.js

```diff
@@ -31,14 +31,16 @@ const defaultProps = {
 
 const PDFView = props => (
     <View style={[styles.flex1, props.style]}>
-        <PDF
-            activityIndicator={<FullScreenLoadingIndicator />}
-            source={{uri: props.sourceURL}}
-            style={[
-                styles.imageModalPDF,
-                getWidthAndHeightStyle(props.windowWidth, props.windowHeight),
-            ]}
-        />
+        <TouchableWithoutFeedback>
```

Comment:
I think it makes more sense to convert the outer View to `TouchableWithoutFeedback` rather than adding another wrapper.

Reply:
I was concerned about messing up the styles, but they seem to work fine with your suggestion. I'll do that, thanks!

Reply:
This is a good point, and it's my bad. I should have caught that on the proposal.